cli/htmlParser.js

```diff
@@ -155,24 +159,21 @@ function processTaglessChild(dom, inPre) {
          of raw text that was in the parent node, so we just need to make
          a file for this and put the contents in it.
          */
-        contents = dom.value;
+
+
+        var dummyNode = parser.parse(' ');
+        dummyNode.childNodes = [dom];
+        var contents = serializer.serialize(dummyNode);
```

Comment:
This isn't using the new parse5 serialization option to not escape the string as I expected. Wouldn't that remove the need for your string replacement below?